.github/workflows/ci_pr.yml

```diff
@@ -70,9 +68,9 @@ jobs:
         curl -sSf --retry 5 -o /tmp/python-${{ matrix.python-version }}.tar.bz2 https://dcrdata.blob.core.windows.net/python/python-${{ matrix.python-version }}.tar.bz2
         sudo tar xjf /tmp/python-${{ matrix.python-version }}.tar.bz2 --directory /
         #
-        # The virtual environments for 2.6 and 3.4 have dependencies on OpenSSL 1.0, which is not available beyond Ubuntu 16. We use this script to patch the environments.
+        # The virtual environments have dependencies on old versions of OpenSSL (e.g 1.0/1.1) which are not available on Ubuntu 24. We use this script to patch the environments.
         #
-        if [[ "${{ matrix.python-version }}" =~ ^2\.6|3\.4$ ]]; then
+        if [[ "${{ matrix.use_virtual_environment}}" == "true" ]]; then
```

Comment:
I decided to patch all of them, although 2.7 does not need it, to avoid listing each version. Patching 2.7 is not an issue.